openapi/openapi.yaml

```
@@ -49,381 +57,629 @@ components:
+    description: >-
+      Use reports operations to retrieve summary information about
+      your customers, subscriptions, transactions, and more.
+  - name: Data exports
+    description: |-
+      Use data export operations to manage the export of resource
+      data, such as: transactions, customers, subscriptions, invoices,
+      invoice item data, or revenue audit. Common data export use
+      cases are: accounting, data analysis, reporting, or importing
+      into other databases.
+
+      For detailed information on Rebilly resources, see
+      [Resources](https://www.rebilly.com/docs/dev-docs/concepts/#resources).
   - name: Organization data exports
-    description: |
-      Organization data exports allows you to export your organization's data which you may want to do as a business continuity practice.
-      In addition, it allows you to export data if you decide to migrate to a different payments solution.
+    description: |-
+      Use organization data exports to export your organization's data
+      which you may want to do as a business continuity practice.
+      These may also be used to to export data if you decide to
+      migrate to a different payments solution.
   - name: Rebilly API
-    description: |
-      The Rebilly API is the main API commonly integrated by 3rd parties.  You may see
-      the [Rebilly API Documentation here](https://api-reference.rebilly.com).
+    description: |-
+      The Rebilly API is a collection of the most commonly used
+      endpoints and operations.
+
+      Every action in the Rebilly
+      [frontend GUI application](https://app.rebilly.com/) is
+      supported by an API. These APIs are documented and you can use
+      them to automate any workflow.
+
+      For more information, see the
+      [Rebilly API](https://api-reference.rebilly.com).
   - name: Reports API
-    description: |
-      The Rebilly Reporting API is currently experimental.  You may see
-      the [Reports API Documentation here](https://reports-api-docs.rebilly.com).
+    description: |-
+      The Reports API is a collection of report related
+      endpoints and operations. This API is currently experimental.
+
+      Every action in the Rebilly
+      [frontend GUI application](https://app.rebilly.com/) is
+      supported by an API. These APIs are documented and you can use
+      them to automate any workflow.
+
+      For more information, see the
+      [Reports API](https://reports-api-docs.rebilly.com).
   - name: Storefront API
-    description: |
-      The Rebilly Storefront API is currently experimental.  You may see
-      the [Storefront API Documentation here](https://storefront-api-docs.rebilly.com/).
-  - name: Users API
-    description: >
-      The Rebilly User API is primarily for our GUI apps.  You may see
+    description: |-
+      The Storefront API is a collection of transaction related
```

```suggestion
      The Storefront API is a collection of customer related
```

Not sure if it is only transaction related. It is a customer scoped API